Src/PerformanceCollector/Unit.Tests/PerformanceCollectorModuleTests.cs

```diff
@@ -281,25 +283,25 @@ public void CustomCountersSanitizingTest()
                     Assert.AreEqual(@"CategoryNameTwo - CounterNameTwo", collector.Counters[3].Item1.ReportAs);
 
                     Assert.AreEqual(@"\CategoryName3\CounterName3", collector.Counters[4].Item1.OriginalString);
-                    Assert.AreEqual(@"CategoryName - CounterName", collector.Counters[4].Item1.ReportAs);
+                    Assert.AreEqual(@"CategoryName3 - CounterName3", collector.Counters[4].Item1.ReportAs);
 
                     Assert.AreEqual(@"\CategoryName4\CounterName4", collector.Counters[5].Item1.OriginalString);
-                    Assert.AreEqual(@"Counter", collector.Counters[5].Item1.ReportAs);
+                    Assert.AreEqual(@" Counter 4", collector.Counters[5].Item1.ReportAs);
 
                     Assert.AreEqual(@"\CategoryName5\CounterName5", collector.Counters[6].Item1.OriginalString);
-                    Assert.AreEqual(@"Counter", collector.Counters[6].Item1.ReportAs);
+                    Assert.AreEqual(@" Counter5", collector.Counters[6].Item1.ReportAs);
 
                     // unicode-only reportAs values are converted to "random" strings
                     Assert.AreEqual(@"\Категория6\Счетчик6", collector.Counters[7].Item1.OriginalString);
-                    Assert.AreEqual(@"Performance counter A", collector.Counters[7].Item1.ReportAs);
+                    Assert.AreEqual(@"Только юникод первый", collector.Counters[7].Item1.ReportAs);
 
                     Assert.AreEqual(@"\Категория7\Счетчик7", collector.Counters[8].Item1.OriginalString);
-                    Assert.AreEqual(@"Performance counter B", collector.Counters[8].Item1.ReportAs);
+                    Assert.AreEqual(@"Только юникод второй", collector.Counters[8].Item1.ReportAs);
```

Comment:
Just curious why these are 1st and 2nd unicode strings when the test sequence is at 6 & 7? ;) (no need to change it)